lib/bgs/base.rb

```diff
@@ -155,6 +155,12 @@ def client
       @client ||= Savon.client(options)
     end
 
+    def healtcheck?
+      request(wsdl) ? true : false
```

Comment:
`wsdl` returns a URL, but `request` expects the first arg to be a `method`? I guess `method` doesn't mean HTTP verb here, right?

Reply:
From the way I read the code "method" appears to be the url as it's using in client.call(method) below but I'll let Charlie correct me if I'm wrong.

Reply:
Changed